pagerduty/table_pagerduty_on_call.go

```diff
@@ -0,0 +1,90 @@
+package pagerduty
+
+import (
+	"context"
+
+	"github.com/PagerDuty/go-pagerduty"
+
+	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+)
+
+//// TABLE DEFINITION
+
+func tablePagerDutyOnCall(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "pagerduty_on_call",
+		Description: "An on-call represents a contiguous unit of time for which a User will be on call for a given Escalation Policy and Escalation Rules.",
+		List: &plugin.ListConfig{
+			Hydrate: listPagerDutyOnCalls,
+		},
+		Columns: []*plugin.Column{
+			{

```

Comment:
Please update the column order as below - 
`escalation_level`
`start`
`end`
`escalation_policy`
`schedule`
`user`
JSON type columns should be last.